components/automate-cli/cmd/chef-automate/certRotate.go

```diff
@@ -133,21 +142,24 @@ const (
 	sudo systemctl stop hab-sup.service
 	echo "y" | sudo cp /tmp/%s /hab/user/automate-ha-%s/config/user.toml
 	sudo systemctl start hab-sup.service`
+
+	IP_V4_REGEX = `(\b25[0-5]|\b2[0-4][0-9]|\b[01]?[0-9][0-9]?)(\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)){3}`
 )
 
+var readFileFromOs = ioutil.ReadFile // nosemgrep
```

Comment:
Change this to an Interface which you will implement to call actual function here, and mock function in Test

Reply:
done